lizmap/modules/lizmap/controllers/service.classic.php

```diff
@@ -333,12 +336,18 @@ protected function getServiceParameters()
                             && count($data->ids) > 0
                         ) {
                             $selections[] = $data->typename.':'.implode(',', $data->ids);
+                            $feature_ids[] = implode(',', array_map(fn ($c) => $data->typename.'.'.$c, $data->ids));
```

Comment:
if we want to support PHP < 7.4, we should not use the `fn` keyword and this syntax of "short inline function"